eth/v1alpha1/validator.proto

```diff
@@ -135,6 +135,16 @@ service BeaconNodeValidator {
         };
     }
 
+    // MultipleValidatorStatus returns a list of validator statuses on the current epoch.
+    // The request can specify a list of validator public keys.
+    //
+    // Returns a list of ValidatorStatusResponses.
+    rpc MultipleValidatorStatus(MultipleValidatorStatusRequest) returns (MultipleValidatorStatusResponse) {
```

Comment:
Why both `ValidatorStatus` and `MultipleValidatorStatus`?  We should check whether this follows the current API pattern

Reply:
I thought it would be cleaner this way since the `ValidatorStatusResponse` message type is also used here:
https://github.com/prysmaticlabs/ethereumapis/blob/df460bd3d84be4ff3df0658395c7dc9d2a7e7b3d/eth/v1alpha1/validator.proto#L244

Changing `ValidatorStatusResponse` message type to
```
+message ValidatorStatusMetadata {
+    // A list of block numbers of the Ethereum proof-of-work chain
+    // where the deposit for the validator was included.
+    repeated uint64 eth1_deposit_block_numbers = 7;
+
+    // A list of slots in the beacon chain in which the validator's
+    // deposit was included in a block.
+    repeated uint64 deposit_inclusion_slots = 8;
+
+    // A list of epochs in the beacon chain in which the validator
+    // is determined as active.
+    repeated uint64 activation_epochs = 9;
+
+    // A list of positions in the activation queue of pending validators.
+    repeated uint64 position_in_activation_queues = 10;
+}
+
 message ValidatorStatusResponse {
    // The corresponding validator status.
    ValidatorStatus status = 1;

    // The block number of the Ethereum proof-of-work chain
	// where the deposit for the validator was included.
    uint64 eth1_deposit_block_number = 2;

    // The slot in the beacon chain in which the validator's
    // deposit was included in a block.
    uint64 deposit_inclusion_slot = 3;

    // The epoch in the beacon chain in which the validator
    // is determined as active.
    uint64 activation_epoch = 4;

    // The position in the activation queue of pending validators.
    uint64 position_in_activation_queue = 5;
+
+    // All list of validator status metadata mapped 1-to-1 with the public keys in the request.
+    repeated ValidatorStatusMetadata statuses = 6;
 }
```
makes the API seem super confusing.